tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -36,9 +31,94 @@ In most scenarios, you are recommended to use private endpoint connection over V
 
 ## Set up a private endpoint with AWS
 
-This section describes how to set up a private endpoint with AWS PrivateLink.
+This section describes how to set up a private endpoint with AWS PrivateLink for a Serverless Tier cluster and a Dedicated Tier cluster.
+
+### Serverless Tier
 
-In addition to the [prerequisites](#prerequisites), there are 5 steps to set up a private endpoint connection with AWS PrivateLink:
+To connect to your Serverless Tier cluster via a private endpoint, follow these steps:
+
+1. [Choose a TiDB cluster](#step-1-choose-a-tidb-cluster)
+2. [Create an AWS interface endpoint](#step-2-create-an-aws-interface-endpoint)
+3. [Connect to your TiDB cluster](#step-3-connect-to-your-tidb-cluster)
+
+#### Step 1. Choose a TiDB cluster
+
+1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target Serverless Tier cluster to go to its overview page.
+2. Click **Connect** in the upper-right corner. A connection dialog is displayed.
+3. In the **Endpoint Type** drop-down list, select **Private**.
+4. Take a note of **Service Name**, **Availability Zone ID**, and **Region ID**.
+
+    > **Note:**
+    >
+    >  You only need to create one private endpoint per AWS region, which can be shared by all Serverless Tier clusters located in the same region.
+
+#### Step 2. Create an AWS interface endpoint
+
+<SimpleTab>
+<div label="Use AWS Console">
+
+To use the AWS Management Console to create a VPC interface endpoint, perform the following steps:
+
+1. Go to **VPC** > **Endpoints**.
+2. Click **Create Endpoint**.
+
+    The **Create endpoint** page is displayed.
+
+    ![Verify endpoint service](/media/tidb-cloud/private-endpoint/create-endpoint-2.png)
+
+3. Select **Other endpoint services**.
+4. Enter the **Service Name**.
+5. Click **Verify service**.
+6. Select your VPC in the drop-down list. Click the **Additional settings** icon and select the **Enable DNS name** checkbox.
+7. Select the availability zone where your TiDB cluster is located in the **Subnets** area.
```

Comment:
```suggestion
7. In the **Subnets** area, select the availability zone where your TiDB cluster is located, and select the Subnet ID.
```

Reply:
If the Subnect ID is not selected, the following error will be reported
![hXM9PGInAi](https://user-images.githubusercontent.com/79440533/236977993-a688be81-ed32-4cb3-bc77-1c7a60c2359b.jpg)